agent/host.go

```diff
@@ -134,5 +135,52 @@ func createRootNode(g *graph.Graph) (*graph.Node, error) {
 		m.SetField("VirtualizationRole", hostInfo.VirtualizationRole)
 	}
 
+	dmi := dmidecode.New()
+	if err := dmi.Run(); err == nil {
```

Comment:
i would prefer "log warn" explain that the DMI info will not be reported in case of error due to dmidecode error